Lecture_11/Task1/Program.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+        }
+
+        public void Method(IFigure figure)
+        {
+            IFigure[] myArray = new IFigure[5];
+            for (int i = 0; i < myArray.Length; i++)
+            {
+                myArray[i] = figure;
+            }
+        }
+    }
+    public interface IFigure
+    {
+        double CalculateAreaSquare();
+        double CalculateAreaCircle();
+        double CalculateAreaTriangle();
+    }
```

Comment:
this is not a flexible approach. The interface should define general abstraction, but the interface you defined contains string coupling (depends on Square, Circle, Trianle)